src/main/java/org/gridsuite/loadflow/server/LoadFlowController.java

```diff
@@ -170,26 +170,24 @@ public ResponseEntity<List<LimitViolationInfos>> getLimitViolations(
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(result);
     }
 
-    @GetMapping(value = "/limit-types", produces = APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/results/{resultUuid}/limit-types", produces = APPLICATION_JSON_VALUE)
     @Operation(summary = "Get available limit types")
     @ApiResponses(@ApiResponse(responseCode = "200", description = "List of available limit types"))
```

Comment:
Those are not the "available limit types" anymore. More like "Get the limit types present in the result" (and "List of limit types present in the result"). Same remark for the 2 other similar endpoints.